file-formats/srvb/examples/z_aff_example_srvb.srvb.json

```diff
@@ -4,20 +4,15 @@
     "description": "Example SRVB for ABAP File Formats",
     "originalLanguage": "EN"
   },
-  "bindTypeImpl": "Z_AFF_EXAMPLE_SRVB",
-  "bindType": "ODATA",
-  "bindTypeCategory": "0",
-  "bindTypeVersion": "V4",
+  "bindingType": "ODATA V4",
+  "bindingTypeCategory": "ui",
   "services": [
     {
-      "serviceName": "SRVD_HOME",
-      "serviceContent": [
+      "name": "SRVD_HOME",
+      "content": [
```

Comment:
Is it `content` or is is something more specific like `versions` or something else?